service/telemetry.go

```diff
@@ -104,6 +110,7 @@ func (tel *colTelemetry) initOnce(asyncErrorChannel chan<- error, ballastSizeByt
 		zap.String("address", metricsAddr),
 		zap.Int8("level", int8(level)), // TODO: make it human friendly
 		zap.String(semconv.AttributeServiceInstanceID, instanceID),
+		zap.String(semconv.AttributeServiceVersion, collectorVersion),
```

Comment:
```suggestion
		zap.String(semconv.AttributeServiceVersion, version.Version),
```

This is always going to be emitted, so might as well always include the actual version.